.github/workflows/test.yml

```diff
@@ -35,15 +35,20 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Build and export
+      - name: Build docker image
         uses: docker/build-push-action@v5
         with:
           file: resources/images/commander/Dockerfile
           context: .
           tags: bitcoindevproject/warnet-commander:latest
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          outputs: type=docker,dest=/tmp/commander.tar
+          load: true
+
+      - name: Export docker image with tags
+        run: |
+          # This workaround should not be needed...
```

Comment:
This indicates that whilst #523 was working, it was not working as intended.

With this change we now (for sure!) use the CI-built docker image in the CI run...

Reply:
a bit confused as to what's going on here. is #523 good or do we need a fix? can we drop the work around if everything is good?

Reply:
I'm also intrigued